propagation_test.go

```diff
@@ -30,13 +30,20 @@ import (
 	"github.com/opentracing/opentracing-go/ext"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/uber/jaeger-lib/metrics"
 )
 
+func initMetrics() (*metrics.LocalBackend, *Metrics) {
+	b := metrics.NewLocalBackend(0)
+	factory := metrics.NewLocalFactory(b)
```

Comment:
This always made me to question - why are we forced to create local backend if we're calling a dedicated NewLocalFactory?  The backend is not a public API of jaeger-lib/metrics, so I think it can be hidden behind the LocalFactory struct (still exposed for its getter functions).